eth_bridge/eth/contracts/EthProxy.sol

```diff
@@ -61,4 +64,27 @@ contract EthProxy {
         // Send the message to the IC
         terabethiaCore.sendMessage(CANISTER_ADDRESS, payload);
     }
+
+    function send(address recipient, uint256 amount)
```

Comment:
What is this function for? Also does just give arbitrary ownership of all funds in the contract to the owner?